src/webhook/setupGithubWebhooks.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * This is temporary file and  will be removed once exiting event handlers are hooked-in.
+ * 
+ */
+
+import { WebhookContext } from "./types";
+import { Webhooks } from "./webhooks";
+
+export const setupGithubWebhooks = (webhooks: Webhooks) => {
+	webhooks.on(
+		[
+			"issue_comment.created",
+			"issue_comment.edited"
+		],
+		(event: WebhookContext) => {
+			console.log("issue_comment created and edited ", event.id)
```

Comment:
Feels like a Logger instance instead of console.log would be useful.

Reply:
This is temporary one. Will be removed in upcoming PRs.

Reply:
might as well do it now then?

Reply:
I agree, it is better to use a context logger because it adds some useful info, e.g. request id that we can use later for tracing in splunk

Reply:
Now I'm using logger instance.